zulip_bots/zulip_bots/bots/joinme/joinme.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any
+import requests
+import json
+import socket
+
+class JoinmeHandler(object):
+    def usage(self) -> str:
+        return '''
+         A bot that will post a link to joinme call. Mention the bot and it will give out instructions.
+        '''
+
+    def handle_message(self, message: Any, bot_handler: Any) -> None:
+        if 'ZISSHUDBUNIq' not in message['content']:
+            payload = {'client_id': 'fvg6qhpans4z2jryq45crb69', 'scope': 'user_info scheduler start_meeting',
+                       'redirect_uri': 'https://developer.join.me/io-docs/oauth2callback',
+                       'state': 'ZISSHUDBUNIq', 'response_type': 'code'}
+            r = requests.get('https://secure.join.me/api/public/v1/auth/oauth2', params=payload)
+            content = "Please click the link below to log in and authorise Joinme:\n {} \nAnd please copy and " \
+                      "send the url shown in browser after clicking accept. Don't forget to @ me!".format(r.url)
+        else:
+            authorisation_code = message['content']
+            authorisation_code = authorisation_code.split('=')
+            authorisation_code = authorisation_code[1].split('&')
+
+            payload = {"client_id": "fvg6qhpans4z2jryq45crb69", "client_secret": "3BsfF8wRe5",

```

Comment:
You should never include values secret to your account like "client_secret" in your PRs :).